### PR TITLE
Order 1

### PR DESCRIPTION
**Pull request by navacohen90**
is not ready yet. don't review

### CODE REVIEW

**Comment by navacohen90** [comment]
@sarah-hoter @ChagitY please review and merge. thx.

File: controllers/course.controller.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿var config = require('./../../config.json');
+﻿var config = require('./../config.json');
```
Comment (sarah-hoter) [inline controllers/course.controller.js:1]: ./../ is not required, config.json only is supposed to work

File: models/course.model.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+var courseSchema = new Schema({
+    id : Number,
+    courseTypeId : Number,
+    label : String,
+    image : String
+});
+
+
+module.exports = courseSchema;
```
Comment (sarah-hoter) [inline models/course.model.js:12]: what with course price?

**Comment by sarah-hoter** [comment]
@navacohen90 very nice!!

**Pull request merged**